docs/supports-and-hinges/relconnectssurfaceedge.md

```diff
@@ -17,21 +17,27 @@ This object is referring to the edge of StructuralSurfaceMember. Only for this o
+|   Stiffness Fiy \[MNm/rad/m]  |      Double      |                                                                        50                                                                       | yes, if fiy = Flexible or Nonlinear | The flexibility in rotation of the connection around local Y axis                                                                                                                                                                                                                                                                                                                                                  |
+|   Stiffness Fiz \[MNm/rad/m]  |      Double      |                                                                        50                                                                       | yes, if fiz = Flexible or Nonlinear | The flexibility in rotation of the connection around local Z axis                                                                                                                                                                                                                                                                                                                                                  |
+| Function ux | String | Nonlin fun 1 | yes, if ux = Nonlinear | Reference to object [NonlinearFunction](../supports-and-hinges/nonlinearfunction.md). Nonlinear function must be of type Translation |
+| Function uy | String | Nonlin fun 1 | yes, if uy = Nonlinear | Reference to object [NonlinearFunction](../supports-and-hinges/nonlinearfunction.md). Nonlinear function must be of type Translation |
+| Function uz | String | Nonlin fun 1 | yes, if uz = Nonlinear | Reference to object [NonlinearFunction](../supports-and-hinges/nonlinearfunction.md). Nonlinear function must be of type Translation |
+| Function fix | String | Nonlin fun 2 | yes, if fix = Nonlinear | Reference to object [NonlinearFunction](../supports-and-hinges/nonlinearfunction.md). Nonlinear function must be of type Rotation |
+| Function fiy | String | Nonlin fun 2 | yes, if fiy = Nonlinear | Reference to object [NonlinearFunction](../supports-and-hinges/nonlinearfunction.md). Nonlinear function must be of type Rotation |
+| Function fiz | String | Nonlin fun 2 | yes, if fiz = Nonlinear | Reference to object [NonlinearFunction](../supports-and-hinges/nonlinearfunction.md). Nonlinear function must be of type Rotation |
```

Comment:
Something i've overlooked in the previous PR too, the 

> Nonlinear function must be of type Translation
or
> Nonlinear function must be of type Rotation

is something we currently can't easily validate in the SDK...

Reply:
We'll look at adding some kind of validation to the SDK